src/device_trezor/trezor/transport.cpp

```diff
@@ -223,6 +223,11 @@ namespace trezor{
     msg = msg_wrap;
   }
 
+  static void assert_port_number(uint32_t port)
+  {
+    CHECK_AND_ASSERT_THROW_MES(port > 1024 && port < 65535, "Invalid port number: " << port);
```

Comment:
If you mean to reject root reserved ports, 1024 is not in those.

Reply:
correct, fixed